src/GraphQL.Client/GraphQLHttpRequest.cs

```diff
@@ -38,6 +39,9 @@ public virtual HttpRequestMessage ToHttpRequestMessage(GraphQLHttpClientOptions
         {
             Content = new StringContent(serializer.SerializeToString(this), Encoding.UTF8, options.MediaType)
         };
+        message.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("application/graphql+json"));
```

Comment:
`MediaTypeWithQualityHeaderValue` does not accept `application/graphql+json; charset=utf-8`. It throws `System.FormatException : The format of value 'application/graphql+json; charset=utf-8' is invalid`. I added Accept-Charset header that should work the same.